README.md

```diff
@@ -66,13 +65,6 @@
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
@HAMMAS-SALEEM , Dear I really appreciate your cooperation